db/db_options_test.cc

```diff
@@ -117,6 +117,77 @@ TEST_F(DBOptionsTest, GetLatestCFOptions) {
             GetMutableCFOptionsMap(dbfull()->GetOptions(handles_[1])));
 }
 
+TEST_F(DBOptionsTest, SetBytesPerSync) {
+  Options options;
+  options.create_if_missing = true;
```

Comment:
also set 
1. `options.write_buffer_size = (a value larger than 10*kValue.size() by some margin)` to make sure flush won't auto-triggered. 
2. `options.disable_auto_compactions = true`

Same for SetWalBytesPerSync test, but write buffer size should be larger than 20*kValueSize() there.